.travis.yml

```diff
@@ -122,27 +124,35 @@ script:
     # Copy script to testing directory
     - cp integration_new/extra/travis-testing/* tests/
 
-    # Build docker image from docker file
-    - sudo docker build -t $DOCKER_REPOSITORY:pr .
-
-
```

Comment:
Careful with this rename. The deploy section unconditionally pushes the `pr` tag, so this will either cause that step to fail, or worse, cause it to push an old, unrelated, and possibly broken `pr` tag. Maybe you should check the job type in the deploy section as well.